### ISSUE TITLE
Retire in favour of pybufrkit?

### ISSUE THREAD

**Issue by gerritholl** [pytroll/python-bufr#17]
The python-bufr package appears poorly maintained.  The newest commit is nearly seven years old, the most recent issue has seen no activity since January 2016, nearly six years ago.  I'm unable to install it or run the unit tests (there is no `setup.py` or `pyproject.toml`).  A different package, [pybufrkit](https://github.com/ywangd/pybufrkit), appears to have similar aims.  It has 45 passing unit tests with 82% test coverage.  It is [publicly recommended by DWD](https://www.dwd.de/EN/aboutus/it/functions/Teasergroup/bufr.html) and sees some maintenance (last version released less than three months ago, 12 May 2021).

Is there some unique functionality in pytroll-bufr that is missing in pybufrkit, or another reason to continue maintaining both packages?

**Comment by mraspaud** [comment]
I agree that this should be retired. This was implemented by Rune who doesn't work with this any more since 2013. No one has taken over, and from what I see, the most recent change in the forks is from 2015.
